jekyll/_cci2/orb-author.md

```diff
@@ -17,104 +17,125 @@ version:
 
-    **Ensure the context is restricted**
-    <br/>
     Restrict a context by navigating to _Organization Settings > Contexts_.
-    <br/><br/>
+
     After completing your orb, you should see a new context called `orb-publishing`. Click into `orb-publishing` and add a _Security Group_. Use Security Groups to limit access to users that are allowed to trigger jobs. Only these users will have access to the private [Personal API Token]({{site.baseurl}}/2.0/managing-api-tokens/).
-    <br/><br/>
+
     For more information, see the [Contexts]({{site.baseurl}}/2.0/contexts/#restricting-a-context) guide.
     {: class="alert alert-warning"}
 
 1. **Push the changes up to Github.**
-<br>
+
     During the setup process, the `orb init` command takes steps to prepare your automated orb development pipeline. The modified template code produced by the CLI must be pushed to the repository before the CLI can continue and automatically follow your project on circleci.com. Run the following command from a separate terminal when prompted to do so, substituting the name of your default branch:
     ```shell
     git push origin <default-branch>
     ```
     Once complete, return to your terminal and confirm the changes have been pushed.
 
-1. **Complete and write your orb.**
-<br>
-    The CLI will finish by automatically following your new orb project on CircleCI and generating the first development version `<namespace>/<orb>@dev:alpha` for testing (a hello-world sample).
+1. **Complete the setup**
 
-    You will be provided with a link to the project building on CircleCI where you can view the validation, packing, testing, and publication process.
-    You should also see the CLI has automatically migrated you into a new development branch, named `alpha`.
+    Once the changes have been pushed, return to your terminal and continue the setup process. The CLI will now automatically follow the project on CircleCI, and attempt to trigger a pipeline to build and test your orb with sample code.
 
-    From your new branch you are now ready to make and push changes. From this point on, on every commit, your orb will be packed, validated, tested (optional), and can be published.
+    You will be provided with a link to the project building on CircleCI where you can view the full pipeline.
+    You should also see the CLI has automatically migrated you into a new development branch, named `alpha`. You can use any branch naming you would like, you do not need to exclusively develop on `alpha`.
 
-    When you are ready to deploy the first major version of your orb, find information on deploying changes with semver versioning in the [Orb Publishing Process]({{site.baseurl}}/2.0/creating-orbs/) guide.
+    Because we are making use of [dynamic configuration]({{site.baseurl}}/2.0/dynamic-config/), we must first enable this feature. You will receive an error on your first pipeline that will state that this feature is not enabled.
```

Comment:
I feel like this is another step?

Reply:
Ya i was trying to be sneaky. Ill break it up lol